app.py

```diff
@@ -4,11 +4,11 @@
 
 
 app = Flask(__name__)
-app.config.from_object('config')
-app.register_blueprint(api_bp, url_prefix='/api')
-app.register_blueprint(stocks_bp, url_prefix='/stocks')
+app.config.from_object("config")
+app.register_blueprint(api_bp, url_prefix="/api")
+app.register_blueprint(stocks_bp, url_prefix="/stocks")
 
-@app.route('/')
+@app.route("/")
```

Comment:
Either nuke this or move healthcheck here, unless you want this for some reason

Reply:
moved healthcheck